packages/circuit-ui/components/Button/Button.tsx

```diff
@@ -233,13 +233,13 @@ const tertiaryStyles = ({
   `;
 };
 
-const sizeStyles = ({ theme, size = 'mega' }: ButtonProps & StyleProps) => {
+const sizeStyles = ({ theme, size = 'giga' }: ButtonProps & StyleProps) => {
   const sizeMap = {
     kilo: {
       padding: `${theme.spacings.bit} calc(${theme.spacings.mega} - ${BORDER_WIDTH})`,
     },
-    mega: {
-      padding: `${theme.spacings.byte} calc(${theme.spacings.giga} - ${BORDER_WIDTH})`,
+    giga: {
+      padding: `calc(${theme.spacings.kilo} - ${BORDER_WIDTH}) calc(${theme.spacings.giga} - ${BORDER_WIDTH})`,
```

Comment:
@connor-baer I'm interested about your thoughts here–we didn't find a way to avoid the `calc()` in every component

Reply:
I figured it would be this way for the Button — all good 👍🏻